### DISCUSSION TITLE
keymap e

### DISCUSSION THREAD

**Discussion by tharindu899** [LazyVim/LazyVim#1550] [Q&A]
<leader>e | Explorer NeoTree (root dir)

how to keymap this ctrl+n

**Comment by dpetka2001** [comment]
```lua
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    keys = {
      { "<C-n>", "<leader>fe", desc = "Explorer NeoTree (root dir)", remap = true },
    },
  },
}
```
Put this snippet in a file in `~/.config/nvim/lua/plugins/` directory.

**Reply by dpetka2001** [reply]
You should also read the [docs](https://www.lazyvim.org/plugins/editor#neo-treenvim) and see how the keymaps are declared. This way you'd know how to do it yourself.
